render/data/spec/data-spec.js

```diff
@@ -193,6 +193,18 @@ describe("data", function() {
       });
     });
 
+    describe("kebabCase", function() {
```

Comment:
Additional cases worth adding?
* "getThingUUID"
* "ThingClass"

Consider this logic?
```
'getThingUUID'.replace(/B[A-Z]+/g, s => '-' + s.toLowerCase() )
"get-item-uuid"
```

Or avoid the clutter of having to implement the method + test cases and, instead, use something like https://github.com/blakeembrey/change-case?

Reply:
I added your fix. I don't like to introduce libraries for small things because "dependency hell" is already bad enough as is. In fact, there is a large amount of work necessary to upgrade existing dependencies.

Reply:
[True dat](https://npmgraph.js.org/?q=change-case).